frontend/src/Components/Algorithms/Algorithm4.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import Base from '../../Layouts/Base'
+import Content from '../../Layouts/Content'
+import Scenario from './Scenario'
+
+class Algorithm4 extends Component {
+  render() {
+    return (
+      <div >
+        <Base 
+        content={
+        <div>
+        <Content text={'Scenario'} compo={<Scenario />}/> 
+        <br/><Content text={'Results'}/>        </div>
```

Comment:
I know little about react, but I think we are supposed to format the embedded html code as well.